tests/test_pandas_extension.py

```diff
@@ -144,6 +144,12 @@ def type_fails_on_stable_pandas(test_type):
 ]
 
 
+@pytest.fixture(params=[True, False])
+def box_in_series(request):
```

Comment:
Pandas defines this in `pandas/tests/extension/conftest.py` Doesn't pytest automatically pick it up? Or perhaps this is just for compatibility with 0.23.x as well as master?

Reply:
No, this was not picked up automatically. We could probably import it instead of redefining it here. I'm unsure about how py.test would behave with the other fixtures in regard to importing `pandas/tests/extension/conftest.py`. 

For 0.23.x, this is not yet needed as the tests there don't require the fixture and we're not yet overriding a test here that uses the fixture.